src/app/movieList/movieCard/movieCard.component.css

```diff
@@ -0,0 +1,24 @@
+.row {
+  margin-left: 10px;
+  margin-right: 10px;
+}
+
+.row:after {
+  content: normal;
+}
+
+.card {
+  height: 27.875em;
```

Comment:
I said you should delete fixed height, it brokes markup on wide displays 
See - 
![image](https://user-images.githubusercontent.com/12671147/43592127-9312d400-967d-11e8-8d41-486c975c33e2.png)